src/main/java/seedu/us/among/logic/commands/AddCommand.java

```diff
@@ -56,6 +51,6 @@ public CommandResult execute(Model model) throws CommandException {
     public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof AddCommand // instanceof handles nulls
-                && toAdd.equals(((AddCommand) other).toAdd));
+                        && toAdd.equals(((AddCommand) other).toAdd));
```

Comment:
I think this extra indentation is not necessary?

Reply:
newline indentation must be 2 tabs wide, some checkstyle error flagged it so no choice